index.d.ts

```diff
@@ -20,13 +20,13 @@ declare module 'connected-react-router' {
   export const LOCATION_CHANGE: string
   export const CALL_HISTORY_METHOD: string
 
-  export function push(path: Path, state?: LocationState): Action
-  export function push(location: LocationDescriptorObject): Action
-  export function replace(path: Path, state?: LocationState): Action
-  export function replace(location: LocationDescriptorObject): Action
-  export function go(n: number): Action
-  export function goBack(): Action
-  export function goForward(): Action
+  export function push(path: Path, state?: LocationState): RouterAction;
```

Comment:
So, we no longer need to import Action from 'redux'. Can you help remove it?

Reply:
Ah, yep, I missed that, just added a commit to clean it up